docs/topics/api/signing.rst

```diff
@@ -131,6 +132,9 @@ property.
             }
 
     :>json active: version is active.
+    :>json automated_signing:
+        if the version will be signed automatically, if this is false it will
```

Comment:
I know the other fields use incomplete sentences but I think you should make it complete, like: If true, the version will be signed automatically.